play-ahc-ws-standalone/src/main/scala/play/api/libs/ws/ahc/StandaloneAhcWSRequest.scala

```diff
@@ -57,19 +57,25 @@ case class StandaloneAhcWSRequest(
   override def contentType: Option[String] = this.headers.get(HttpHeaders.Names.CONTENT_TYPE).map(_.head)
 
   override lazy val uri: URI = {
```

Comment:
This `uri` is not used when firing a request. It's just used in tests. It's looks like it was just added to have a nice way to see what url was set on `StandaloneAhcWSRequest`.